Frameworks/UIKit/NSString+UIKitAdditions.mm

```diff
@@ -41,6 +41,18 @@ void NSStringForceinclude() {
 
 @implementation NSString (UIKitAdditions)
 
+static CTTextAlignment __UITextAlignmentToCTTextAlignment(UITextAlignment alignment) {
```

Comment:
unfortunately the reference platform has different values for UI/CT right/center alignment, which our internal test app (and presumably outside developer apps) depend on the values of.